exporter/prometheusexporter/config.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2019, OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package prometheusexporter
+
+import (
+	"github.com/open-telemetry/opentelemetry-service/models"
+	prometheus_golang "github.com/prometheus/client_golang/prometheus"
```

Comment:
nit: the standard that we tried to keep is to have the imports from the repo it self on the bottom. That said I think we should consider any ordering/organization that can be achieve automatically with most editors.